src/platform/site-wide/mega-menu/mega-menu-link-data-for-authenticated-users.json

```diff
@@ -1,7 +1,7 @@
 [
     {
         "title": "My VA",
-        "href": "/dashboard/",
+        "href": "/my-va/",
```

Comment:
this mega-menu config is only used on the new site so this doesn't require conditional logic